src/git/commit_filter_test.go

```diff
@@ -41,74 +41,143 @@ func TestCommitFilter_Commits(t *testing.T) {
 	for _, tc := range []struct {
 		name            string
 		opts            []git.CommitFilterOptionFunc
-		commits         []git.Commit
 		expectedCommits []git.Commit
 	}{
 		{
 			name: "Included_Folders_All_Excluded_None",
 			opts: []git.CommitFilterOptionFunc{},
 
-			commits:         allCommits,
 			expectedCommits: allCommits,
 		},
 		{
 			name: "Include_Folder1",
 			opts: []git.CommitFilterOptionFunc{
 				git.IncludedDirs("folder1"),
 			},
-			commits: allCommits,
 			expectedCommits: []git.Commit{
 				singleFileFolder1, threeFilesFolders, rootAndFolder,
 			},
 		},
+		{
+			name: "Include_folder1/single-file-on-folder1",
```

Comment:
nit: perhaps a more descriptive name like `Include commits that only modifies IncludedFiles`